### PR TITLE
Prioritize user properties over modeline properties.

### PR DESCRIPTION
**Pull request by doru1004**
This patch ensures that integration properties specified via command line overwrite properties specified via modeline.

**Release Note**
```release-note
Prioritize user-specified integration properties over modeline properties with the same name.
```

### CODE REVIEW

File: deploy/resources.go [hunk 1]
```diff
@@ -147,9 +147,9 @@ var assets = func() http.FileSystem {
...
...c\x0d\xb8\x6d\x78\xe5\x29\x75\x42\xcb\x49\x1f\x8b\xe0\x38\x1d\xeb\x4f\xa9\x95\x70\x7e\x6e\x85\x76\xb2\xfa\x46\xe0\x84\xe1\xd7\x95\x70\x1e\xbc\xcc\xb0\xca\x88\x4a\x12\x7c\x0d\x08\x93\x70\x14\xd8\x68\x2c\x05\xd6\xa5\xe3\x06\x84\x36\x7e\x85\xb6\x3d\x91\xac\x0e\x85\x25\xc2\xe3\xa8\x73\xa2\xfc\x88\xa5\x10\x91\x9f\xf9\x44\xf1\x89\x04\xce\xf9\xa4\xf8\x8e\x48\x4a\x9a\x6a\x2a\x37\xc2\xd5\xe7\x93\xbf\x1d\xce\x19\x3a\xd7\x79\xbe\xe0\x60\x14\xb9\x2a\x32\xa1\xc1\xa2\x48\xf8\x73\xa0\xf2\x55\x4a\x06\x65\x2c\xb8\x08\x4e\xd0\x0b\xa9\x1c\x88\x85\x29\xba\x26\xb0\x24\xcb\x9d\x04\xdb\x49\x7b\x04\x69\x8b\xc2\x75\x05\xa8\x23\x06\x87\xc5\xf5\x59\xc3\x9a\xc1\xaf\x5c\xc9\xfb\xe7\xe1\x72\xec\x27\x3a\x70\x29\x7d\xc4\x2e\xbf\x0f\x68\x9c\x87\xef\xe1\x96\x30\xb7\x05\x9e\xc3\x47\xa1\x1c\x9e\xc3\x67\xbd\xd6\x66\xf3\x34\x8c\xba\xc7\xee\xc7\x23\x77\xb3\xac\xbe\xc3\xa8\x3e\xd6\xa8\x11\x7b\xc2\xde\xdd\x23\xdc\x51\x97\x79\x76\x9c\x41\x7e\x74\x56\x7b\x3c\x7f\xe7\x8f\x7b\x7a\x5d\x2a\x7f\xee\x73\xec\x4d\x3b\x48\x6a\x41\xe1\xe0\xd6\xee\x23\xa8\xb7\x42\xe5\x2b\xf1\x76\x77\xaf\xfc\x5a\x29\x7c\x66\xd4\x78\x1c\x8e\xbb\x61\x32\x01\x6f\xcb\xe3\xd5\xce\x1b\x4b\x16\x18\xee\xfc\x3f\x00\x00\xff\xff\xd9\xac\x86\xa4\xa4\x39\x00\x00"),
```
Comment (astefanutti) [inline deploy/resources.go:152]: It doesn't seem like it is related to that PR.

**Review by astefanutti — APPROVED** [review]

File: deploy/resources.go (hunk 1 above)
Comment (doru1004) [reply deploy/resources.go:152]: This was updated after running make. I've seen this happening a few times. Was this file out of date in master?

Comment (astefanutti) [reply deploy/resources.go:152]: Yes, I think it has already been updated in master by the time you branched it off.

Comment (doru1004) [reply deploy/resources.go]: I just rebased and that update went away! :)

Comment (astefanutti) [reply deploy/resources.go]: 👍🏼

**Comment by astefanutti** [comment]
Thanks!

**Pull request merged**